tools/generators/pbxproject_prefix/src/Generator/Arguments.swift

```diff
@@ -0,0 +1,39 @@
+import ArgumentParser
+import Foundation
+import GeneratorCommon
+
+extension Generator {
+    struct Arguments: ParsableArguments {
+        @Argument(
+            help: """
+Path to where the 'pbxproject_prefix' 'PBXProj' partial should be written.
+""",
+            transform: { URL(fileURLWithPath: $0, isDirectory: false) }
+        )
+        var outputPath: URL
+
+        @Argument(help: "Absolute path to the Bazel workspace.")
+        var workspace: String
+
+        @Argument(
+            help: """
+Path to a file that contains the absolute path to the Bazel execution root.
+""",
```

Comment:
It's because the path is generated during the execution phase, so we can't pass the value itself as an argument. This is because we can't get absolute paths in the analysis phase.